docs/core_concepts/15_authentification/index.mdx

```diff
@@ -0,0 +1,101 @@
+import DocCard from '@site/src/components/DocCard';
+
+# Authentification
+
+Windmill provides flexible authentication options to ensure secure access to the platform. Users can authenticate through Single Sign-On (SSO) with a restricted domain or with a simple e-mail if added manually by superadmins.
+
+<div class="grid grid-cols-2 gap-6 mb-4">
+	<DocCard
+    	title="Roles and Permissions"
+    	description="Find out about the roles within a Windmill instance and their respective permissions."
+    	href="/docs/core_concepts/roles_and_permissions"
+    />
+</div>
+
+## Restricted Domain Authentication
+
+Windmill supports authentication [through SSO](../../misc/2_setup_oauth/index.md) for users with email addresses from a restricted domain. This allows organizations to control access to Windmill based on their domain policy. Users with email addresses from the authorized domain can authenticate seamlessly using their SSO credentials.
+
+To enable restricted domain authentication, an administrator can configure the authorized domain in Windmill's settings. Once configured, users with email addresses from the authorized domain will be able to log in using their SSO provider.
```

Comment:
replace "in Windmill Settings" by, "In the OAuth configuration by setting 'allowed_domains' to the desired domains (e.g: 'windmill.dev' to accept only Google/Microsoft logins with a xxx@windmill.dev address)